src/time/src/mcp_server_time/server.py

```diff
@@ -39,10 +39,18 @@ def get_local_tz(local_tz_override: str | None = None) -> ZoneInfo:
     if local_tz_override:
         return ZoneInfo(local_tz_override)
```

Comment:
Suggestion to add try/catch if the local override results in an exception, plus it looks like McpError expects an object with a message and not a string:
```suggestion
    if local_tz_override:
        try:
            return ZoneInfo(local_tz_override)
        except Exception as e:
            error_data = ErrorData(
                code=INVALID_PARAMS,
                message=f"Invalid timezone: {str(e)}"
            )
            raise McpError(error_data)
```

Reply:
You will also need to add ErrorData and INVALID_PARAMS fom mcp.types, for example:

`from mcp.types import Tool, TextContent, ImageContent, EmbeddedResource, ErrorData, INVALID_PARAMS`